vdirsyncer/storage/google.py

```diff
@@ -94,6 +94,11 @@ def _save_token(token):
 class GoogleCalendarStorage(dav.CaldavStorage):
     __doc__ = '''Google calendar.
 
+    .. note:: You have to poperly set up ``collections``, otherwise errors like
```

Comment:
I think https://github.com/pimutils/vdirsyncer/pull/466 would fix this, so you can take this paragraph out.